tag_poses:
- frame_id: map
  id: 0
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 0.300
  z: -1.450
- frame_id: map
  id: 1
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 0.300
  z: -1.450
- frame_id: map
  id: 2
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 0.300
  z: -1.450
- frame_id: map
  id: 3
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 0.300
  z: -1.450
- frame_id: map
  id: 4
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 0.300
  z: -1.450
- frame_id: map
  id: 5
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 0.300
  z: -1.450
- frame_id: map
  id: 6
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 0.300
  z: -1.450
- frame_id: map
  id: 7
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 0.600
  z: -1.450
- frame_id: map
  id: 8
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 0.600
  z: -1.450
- frame_id: map
  id: 9
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 0.600
  z: -1.450
- frame_id: map
  id: 10
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 0.600
  z: -1.450
- frame_id: map
  id: 11
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 0.600
  z: -1.450
- frame_id: map
  id: 12
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 0.600
  z: -1.450
- frame_id: map
  id: 13
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 0.600
  z: -1.450
- frame_id: map
  id: 14
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 0.900
  z: -1.450
- frame_id: map
  id: 15
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 0.900
  z: -1.450
- frame_id: map
  id: 16
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 0.900
  z: -1.450
- frame_id: map
  id: 17
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 0.900
  z: -1.450
- frame_id: map
  id: 18
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 0.900
  z: -1.450
- frame_id: map
  id: 19
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 0.900
  z: -1.450
- frame_id: map
  id: 20
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 0.900
  z: -1.450
- frame_id: map
  id: 21
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 1.200
  z: -1.450
- frame_id: map
  id: 22
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 1.200
  z: -1.450
- frame_id: map
  id: 23
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 1.200
  z: -1.450
- frame_id: map
  id: 24
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 1.200
  z: -1.450
- frame_id: map
  id: 25
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 1.200
  z: -1.450
- frame_id: map
  id: 26
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 1.200
  z: -1.450
- frame_id: map
  id: 27
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 1.200
  z: -1.450
- frame_id: map
  id: 28
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 1.500
  z: -1.450
- frame_id: map
  id: 29
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 1.500
  z: -1.450
- frame_id: map
  id: 30
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 1.500
  z: -1.450
- frame_id: map
  id: 31
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 1.500
  z: -1.450
- frame_id: map
  id: 32
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 1.500
  z: -1.450
- frame_id: map
  id: 33
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 1.500
  z: -1.450
- frame_id: map
  id: 34
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 1.500
  z: -1.450
- frame_id: map
  id: 35
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 1.800
  z: -1.450
- frame_id: map
  id: 36
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 1.800
  z: -1.450
- frame_id: map
  id: 37
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 1.800
  z: -1.450
- frame_id: map
  id: 38
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 1.800
  z: -1.450
- frame_id: map
  id: 39
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 1.800
  z: -1.450
- frame_id: map
  id: 40
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 1.800
  z: -1.450
- frame_id: map
  id: 41
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 1.800
  z: -1.450
- frame_id: map
  id: 42
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 2.100
  z: -1.450
- frame_id: map
  id: 43
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 2.100
  z: -1.450
- frame_id: map
  id: 44
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 2.100
  z: -1.450
- frame_id: map
  id: 45
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 2.100
  z: -1.450
- frame_id: map
  id: 46
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 2.100
  z: -1.450
- frame_id: map
  id: 47
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 2.100
  z: -1.450
- frame_id: map
  id: 48
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 2.100
  z: -1.450
- frame_id: map
  id: 49
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 2.400
  z: -1.450
- frame_id: map
  id: 50
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 2.400
  z: -1.450
- frame_id: map
  id: 51
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 2.400
  z: -1.450
- frame_id: map
  id: 52
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 2.400
  z: -1.450
- frame_id: map
  id: 53
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 2.400
  z: -1.450
- frame_id: map
  id: 54
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 2.400
  z: -1.450
- frame_id: map
  id: 55
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 2.400
  z: -1.450
- frame_id: map
  id: 56
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 2.700
  z: -1.450
- frame_id: map
  id: 57
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 2.700
  z: -1.450
- frame_id: map
  id: 58
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 2.700
  z: -1.450
- frame_id: map
  id: 59
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 2.700
  z: -1.450
- frame_id: map
  id: 60
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 2.700
  z: -1.450
- frame_id: map
  id: 61
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 2.700
  z: -1.450
- frame_id: map
  id: 62
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 2.700
  z: -1.450
- frame_id: map
  id: 63
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 3.000
  z: -1.450
- frame_id: map
  id: 64
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 3.000
  z: -1.450
- frame_id: map
  id: 65
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 3.000
  z: -1.450
- frame_id: map
  id: 66
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 3.000
  z: -1.450
- frame_id: map
  id: 67
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 3.000
  z: -1.450
- frame_id: map
  id: 68
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 3.000
  z: -1.450
- frame_id: map
  id: 69
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 3.000
  z: -1.450
- frame_id: map
  id: 70
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 3.300
  z: -1.450
- frame_id: map
  id: 71
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 3.300
  z: -1.450
- frame_id: map
  id: 72
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 3.300
  z: -1.450
- frame_id: map
  id: 73
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 3.300
  z: -1.450
- frame_id: map
  id: 74
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 3.300
  z: -1.450
- frame_id: map
  id: 75
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 3.300
  z: -1.450
- frame_id: map
  id: 76
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 3.300
  z: -1.450
- frame_id: map
  id: 77
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 3.600
  z: -1.450
- frame_id: map
  id: 78
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 3.600
  z: -1.450
- frame_id: map
  id: 79
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 3.600
  z: -1.450
- frame_id: map
  id: 80
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 3.600
  z: -1.450
- frame_id: map
  id: 81
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 3.600
  z: -1.450
- frame_id: map
  id: 82
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 3.600
  z: -1.450
- frame_id: map
  id: 83
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 3.600
  z: -1.450
- frame_id: map
  id: 84
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.200
  y: 3.900
  z: -1.450
- frame_id: map
  id: 85
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.450
  y: 3.900
  z: -1.450
- frame_id: map
  id: 86
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.700
  y: 3.900
  z: -1.450
- frame_id: map
  id: 87
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 0.950
  y: 3.900
  z: -1.450
- frame_id: map
  id: 88
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.200
  y: 3.900
  z: -1.450
- frame_id: map
  id: 89
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.450
  y: 3.900
  z: -1.450
- frame_id: map
  id: 90
  qw: 0.000
  qx: 1.000
  qy: 0.000
  qz: 0.000
  size: 0.075
  x: 1.700
  y: 3.900
  z: -1.450
